### DISCUSSION TITLE
Empty index.html files when running hexo generate

### DISCUSSION THREAD

**Discussion by ravsau** [ppoffice/hexo-theme-icarus#1135] [Q&A]
This is my repo and github actions file: https://github.com/ravsau/ravsau.github.io/blob/main/.github/workflows/hexo.yml

When I run `hexo generate` locally, everything works fine and I get expected output. But when running hexo generate on github actions, I get empty index.html files. Has anyone else faced this issue?

**Comment by ppoffice** [comment]
`https://github.com/ravsau/ravsau.github.io/blob/main/.github/workflows/hexo.yml` access is denied.
